### PR TITLE
download naming changes, added decision matrix download button to dam results

### PR DESCRIPTION
**Pull request by elbfox**
@sythel not totally sure that I added the decision matrix download properly, because I was unable to test it locally. 

Sharon wanted me to include a download button for the decision matrix (DecisionMatrices_All.csv in www/ folder) next to every individual dam results table 2 download button, so this PR addresses that request.

### CODE REVIEW

File: src/dams_mcda/server.R [hunk 1]
```diff
@@ -1483,7 +1483,11 @@ server <- function(input, output, session) {
 			write.csv( round(RawTable, 0), file, row.names = TRUE, quote=TRUE)
 		  }
 		)
-
+		
+		# download for Decision Matrix
+		output[[paste0("DownloadDecisionMatrix", damId, "DecisionMatrices_All")]] <- downloadHandler(
+		  filename = "DecisionMatrices_All.csv"	)
+
```
[the remark this answers is not in the document]
Comment (sythel) [reply src/dams_mcda/server.R]:
```
output[[paste0("DownloadDecisionMatrix", damId, "DecisionMatrices_All")]] <- downloadHandler(
```
-> 
```
output[[paste0("DownloadDecisionMatrix", damId)]] <- downloadHandler(
```

[the remark this answers is not in the document]
Comment (sythel) [reply src/dams_mcda/server.R]: > This is maybe a dumb question but do I need to call it "Download Dam Data"

button identifiers shouldn't have spaces, the "Download Dam Data" is the button text not the identifier

[the remark this answers is not in the document]
Comment (sythel) [reply src/dams_mcda/server.R]: @elbfox line 1488 should be exactly
```
output[[paste0("DownloadDecisionMatrix", damId)]] <- downloadHandler(
```

[the remark this answers is not in the document]
Comment (sythel) [reply src/dams_mcda/server.R]: actually I think theres a little more needed for the download handler I will make a PR into branch Revisions_minor

[the remark this answers is not in the document]
Comment (elbfox) [reply src/dams_mcda/server.R]: ok thank you!

[the remark this answers is not in the document]
Comment (sythel) [reply src/dams_mcda/server.R]: PR #189

**Comment by sythel** [comment]
server live with download working

**Pull request merged**